lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1506,14 +1506,11 @@ def _resource_sku_to_capability(  # noqa: C901
         ](is_allow_set=True, items=[])
         for sku_capability in resource_sku.capabilities:
             name = sku_capability.name
+            vcpus = 0
             if name == "vCPUsAvailable":
                 node_space.core_count = int(sku_capability.value)
-                if resource_sku.family in ["standardLSv2Family"]:
-                    # refer https://docs.microsoft.com/en-us/azure/virtual-machines/lsv2-series # noqa: E501
-                    # NVMe disk count = vCPU / 8
-                    nvme = NvmeSettings()
-                    nvme.disk_count = int(node_space.core_count / 8)
-                    node_space.features.add(nvme)
+            if name == "vCPUs":
```

Comment:
use `elif`. It's clearer, like a `switch`.

Reply:
update